### PR TITLE
Get rid of a warning for the MMSEQS_COLABFOLDSEARCH process

### PR DESCRIPTION
**Pull request by athbaltzis**
---
name: CRG config 
about: A fix in the CRG config for proteinfold
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [ ] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [ ] Add your custom config file to the `conf/` directory
- [ ] Add your documentation file to the `docs/` directory
- [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [ ] Add your custom profile to the `README.md` file in the top-level directory
- [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: conf/pipeline/proteinfold/crg.config [hunk 1]
```diff
@@ -16,11 +16,15 @@ profiles {
...
-                time = '12h'
+        }
+        if (params.mode == 'colabfold') {
+            process {
+                withName: 'MMSEQS_COLABFOLDSEARCH' {
+                    queue = 'mem_512'
+                    memory = "100 GB"
+                    cpus = 8
+                    time = '12h'
+                }
```
Comment (JoseEspinosa) [inline conf/pipeline/proteinfold/crg.config:27]: Maybe add also here the other colabfold process and add a condition for alphafold before line 10?
```suggestion
                }
                withName: 'COLABFOLD_BATCH' {
                    cpus = 1
                    memory = "30 GB"
                    queue = params.use_gpu ? 'gpu' : 'long-sl7'
                    clusterOptions = { ( task.queue == 'gpu' ? '-l gpu=1' : '' ) }
               }

```

**Review by JoseEspinosa — APPROVED** [review]
Maybe we should do the same for the alphafold processes?

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Sorry, I approved it before by mistake

File: conf/pipeline/proteinfold/crg.config (hunk 1 above)
Comment (athbaltzis) [reply conf/pipeline/proteinfold/crg.config:27]: Yes, sure. I will add the change to my commit

**Review by JoseEspinosa — APPROVED** [review]
Awesome @athbaltzis !

**Pull request merged**